mathics/builtin/assignment.py

```diff
@@ -625,6 +627,11 @@ class Definition(Builtin):
     >> Definition[a]
      = a = 2
 
+    >> a = 2;
+    >> ? a
+     = a = 2
+
+
```

Comment:
`Definition` is not related to `Information`:

- `?a` is `Information["a", Rule[LongForm, False]]`
- `??a` is `Information["a", Rule[LongForm, True]]`